etc/kayobe/openstack.yml

```diff
@@ -5,6 +5,10 @@
 # Name of the current OpenStack release. Default is "2023.1".
 #openstack_release:
 
+# Codename of the current OpenStack release. Current release codename is
+# "antelope". This must be manually updated with each release.
+openstack_release_codename: "antelope"
```

Comment:
It was weirdly difficult to word this, and I'm still not exactly happy with it. Suggestions are welcome

Reply:
I like it. I'll start to add the note to other places too

Reply:
> I'd go with something like this:
> 
> ```
> # Codename of the current OpenStack release.
> # NOTE(upgrade): Update to current release codename.
> openstack_release_codename: "antelope"
> ```
> 
> Then in the wiki page we can say to grep for `NOTE(upgrade)`

Done